src/logic/operate.js

```diff
@@ -0,0 +1,39 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  try {
+    const n1 = Big(+numberOne);
+    const n2 = Big(+numberTwo);
+
+    switch (operation) {
+      case '+':
+        result = n1.plus(n2);
+        break;
+
+      case '-':
+        result = n1.minus(n2);
+        break;
+
+      case '×':
+        result = n1.times(n2);
+        break;
+
+      case '÷':
+        result = n1.div(n2);
+        break;
+
+      case '%':
+        result = n1.mod(n2);
+        break;
```

Comment:
- [ ] As stated in the previous review this is a percentage operator and, not a modulus operator. So kindly divide by `100` instead.